parsely/build.gradle

```diff
@@ -26,6 +26,7 @@ android {
     publishing {
```

Comment:
Question (❓): I wonder whether we should migrate to [publish-to-s3-gradle-plugin](https://github.com/Automattic/publish-to-s3-gradle-plugin) at some point and use the `com.automattic.android.publish-to-s3` plugin instead, just like it is done for the rest of our libraries. 🤔

Reply:
That's a good question! When working on publishing of this library, I was considering our S3 instance but I decided to go with Maven Central because:
- Risk delegation
We delegate the risk of "out of service" incidents. If, by some reason, our S3 instance would go down, we would block building apps that use Parse.ly SDK. `mavenCentral` can possibly go down as well, but then it won't be just Parse.ly SDK, but rather vast majority of every dependency clients' projects have 🙂

Other, less important reasons but still:
- Trust 
MavenCentral is well known and trusted service for everyone working with Maven/Gradle projects.
- Repository already included
The `mavenCentral` repository is already included in many projects, there's no reason to ask users to add a new repository.
- Costs
(guessing) We possibly save costs of our S3 instance (less traffic).

I hope that this argumentation to use Maven Central makes sense!

Reply:
That all makes sense to me, thanks for the reply and for considering using the `com.automattic.android.publish-to-s3` already @wzieba ! 🙇